src/modm/utils/type_traits.hpp

```diff
@@ -79,7 +65,21 @@ struct get_callable_signature<F>
 {
     using type = typename impl::mem_fn_helper<decltype(&F::operator())>::type;
 };
+/// @endcond
 
+/**
+ * Deduce function signature type from callable
+ *
+ * Identity for function types
+ * For functions objects the return type and parameters of
+ * operator() are deduced
+ *
+ * e.g. deduces "void(int)" for "[&](int){}"
+ *
+ * @ingroup modm_utils
+ *
+ * \warning Not applicable to overload sets
```

Comment:
```suggestion
 * @warning Not applicable to overload sets
```
Let's not mix "@" and "\" style doxypress command syntax in one comment block.